### ISSUE TITLE
Server crash after killing mobs

### ISSUE THREAD

**Issue by justinschmidtt** [AllTheMods/ATM-8#878]
I've had this happen to me a few times now; when killing mobs (mainly skeletons in caves). Immediately after the mob dies the server crashes which also results in a crash on the client side. 

After restarting the server, the mob is still dead and continue playing. 

Log File:

[2023-02-09-1.log](https://github.com/AllTheMods/ATM-8/files/10700894/2023-02-09-1.log)

Could be related to #876

**Comment by mellamokb** [comment]
Yes!! I've had two crashes, same scenario - killing a skeleton, randomly crashes with the ConcurrentModificationException.
